apps/prairielearn/python/prairielearn/core.py

```diff
@@ -983,10 +973,6 @@ def string_from_numpy(
     if np.isscalar(A):
         assert not isinstance(A, memoryview)
         if presentation_type == "sigfig":
-            if not isinstance(A, np.complexfloating) and isinstance(
```

Comment:
This was almost certainly an oversight, adding this runtime type check. This doesn't make any sense to add here.